examples/event_api_targets/main.tf

```diff
@@ -0,0 +1,59 @@
+terraform {
+  required_version = ">= 0.14.0"
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+locals {
+  environment = "testing"
+}
+
+module "named_event_mapping" {
+  source   = "../../"
+  bus_name = "the-knight-bus"
+  event_patterns = [
+    "event.HogwartsExternal",
+    "event.UnderAgedMagicDone",
+    "event.ProbablyHarryPotter"
+  ]
+
+  targets = {
+    event_api = [
+      {
+        name : "slack"
+        endpoint : "https://hooks.slack.com/services/my/random/key"
+        token : "xoxb-rando-tokenizer"
+        template_vars = {
+          submission_uuid = "$.detail.student_name"
+        }
+
+        template = <<EOF
```

Comment:
I dont know if it makes sense to show this in an example or not, but IRL i think we'd want to reference this with the `file` function and keep the templates in a sep location, yeh?

Reply:
That's probably how I'd want to do it in real life use for sure. We could definitely swap this out in the specs if that'd be cleaner as a real example.